src/_var-reset.scss

```diff
@@ -77,3 +77,4 @@ $breadcrumb-padding-x: $breadcrumb-padding-x / $base-rem-size;
 $breadcrumb-item-padding: $breadcrumb-item-padding / $base-rem-size;
 $kbd-padding-x: $kbd-padding-x / $base-rem-size;
 $kbd-padding-y: $kbd-padding-y / $base-rem-size;
+$close-font-size: $font-size-base * 1.5;
```

Comment:
@fysheets contributed this to make the x button in the modal/status alert look better.